app/api/routes.py

```diff
@@ -0,0 +1,45 @@
+from flask import current_app as app , request , abort, Blueprint
+from flask_bcrypt import generate_password_hash
+from controllers.auth import *
+from utils.decorators import admin_required , admin_manager_required
+
+mongo = app.extensions['mongo']
+role_bp = Blueprint('role',__name__)
```

Comment:
The name of the blueprint shouldn't be role, Name it whatever the purpose of these routes are